Week_04/Lecture_8/assignment3/src/main/java/org/assignment1/assignment3/repository/primaryEmployeeRepository.java

```diff
@@ -0,0 +1,44 @@
+package org.assignment1.assignment3.repository;
+
+import org.assignment1.assignment3.model.employee;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class primaryEmployeeRepository {
+    @Autowired
+    @Qualifier("primaryJdbcTemplate")
+    private JdbcTemplate jdbcTemplate;
```

Comment:
Try to avoid using field injection.

Reply:
I updated the injection method on PrimaryEmployeeRepository also on Secondary to use a setter injection method, Thank you Ms for the suggest.

Reply:
It's better to user contrustor injection

Reply:
Ok Miss, the constructor injection I implemented on the lecture 11 and 12. Thank you for the suggestion